<H2>Description</H2> 
<P>
This track shows a measure of evolutionary conservation in seven species of
the genus <em>Saccharomyces</em> based on a phylogenetic hidden Markov model
(phastCons).  The graphic display shows the alignment projected onto
<em>S. cerevisiae</em>. 
<P>
The genomes were downloaded from:<BR>
<UL>
<LI><em>S. cerevisiae</em> - <A HREF="https://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/" TARGET=_blank>https://downloads.yeastgenome.org/sequence/S288C_reference/genome_releases/</A>
<LI><em>S. paradoxus</em> - <A HREF="ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Spar_contigs.fasta" TARGET=_blank>ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Spar_contigs.fasta</A>
<LI><em>S. mikatae</em> - <A HREF="ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Smik_contigs.fasta" TARGET=_blank>ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Smik_contigs.fasta</A>
<LI><em>S. kudriavzevii</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM6553.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM6553.fsa.gz</A>
<LI><em>S. bayanus</em> - <A HREF="ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Sbay_contigs.fasta" TARGET=_blank>ftp://ftp.broadinstitute.org/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Sbay_contigs.fasta</A>
<LI><em>S. castelli</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM476.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM476.fsa.gz</A>
<LI><em>S. kluyveri</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM479.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM479.fsa.gz</A>
</UL>
</P>

<H2>Display Conventions and Configuration</H2>
<P>
In full and pack display modes, conservation scores are displayed as a
<EM>wiggle track</EM> (histogram) in which the height reflects the 
size of the score. 
The conservation wiggles can be configured in a variety of ways to 
highlight different aspects of the displayed information. 
Click the <A HREF="../goldenPath/help/hgWiggleTrackHelp.html" 
TARGET=_blank>Graph configuration help</A> link for an explanation 
of the configuration options.</P>
<P>
Pairwise alignments of each species to the <em>$organism</em> genome are 
displayed below the conservation histogram as a grayscale density plot (in 
pack mode) or as a wiggle (in full mode) that indicates alignment quality.
In dense display mode, conservation is shown in grayscale using
darker values to indicate higher levels of overall conservation 
as scored by phastCons. </P>
<P>
Checkboxes on the track configuration page allow selection of the
species to include in the pairwise display.  
Configuration buttons are available to select all of the species (<EM>Set 
all</EM>), deselect all of the species (<EM>Clear all</EM>), or 
use the default settings (<EM>Set defaults</EM>).
Note that excluding species from the pairwise display does not alter the
the conservation score display.</P>
<P>
To view detailed information about the alignments at a specific
position, zoom the display in to 30,000 bases or fewer, then click on
the alignment.</P>

<H3>Gap Annotation</H3>
<P>
The <EM>Display chains between alignments</EM> configuration option 
enables display of gaps between alignment blocks in the pairwise alignments in 
a manner similar to the Chain track display.  The following
conventions are used:
<UL>
<LI><B>Single line:</B> No bases in the aligned species. Possibly due to a
lineage-specific insertion between the aligned blocks in the <em>$organism</em> genome
or a lineage-specific deletion between the aligned blocks in the aligning
species.
<LI><B>Double line:</B> Aligning species has one or more unalignable bases in
the gap region. Possibly due to excessive evolutionary distance between 
species or independent indels in the region between the aligned blocks in both
species. 
<LI><B>Pale yellow coloring:</B> Aligning species has Ns in the gap region.
Reflects uncertainty in the relationship between the DNA of both species, due
to lack of sequence in relevant portions of the aligning species. 
</UL></P>

Downloads for data in this track are available:
<UL>
<LI>
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/sacCer3/multiz7way/">Multiz alignments</A> (MAF format), and phylogenetic trees
<LI>
<A HREF="http://hgdownload.soe.ucsc.edu/goldenPath/sacCer3/phastCons7way/">PhastCons conservation</A> (WIG format)
</UL>

<H3>Base Level</H3>
<P>
When zoomed-in to the base-level display, the track shows the base 
composition of each alignment. The numbers and symbols on the Gaps
line indicate the lengths of gaps in the <em>$organism</em> sequence at those 
alignment positions relative to the longest non-<em>$organism</em> sequence. 
If there is sufficient space in the display, the size of the gap is shown. 
If the space is insufficient and the gap size is a multiple of 3, a 
&quot;*&quot; is displayed; other gap sizes are indicated by &quot;+&quot;.</P>
<P>
Codon translation is available in base-level display mode if the
displayed region is identified as a coding segment. To display this annotation,
select the species for translation from the pull-down menu in the Codon
Translation configuration section at the top of the page. Then, select one of
the following modes:
<UL>
<LI> 
<B>No codon translation:</B> The gene annotation is not used; the bases are
displayed without translation. 
<LI>
<B>Use default species reading frames for translation:</B> The annotations from the genome
displayed 
in the <EM>Default species to establish reading frame</EM> pull-down menu are used to
translate all the aligned species present in the alignment. 
<LI>
<B>Use reading frames for species if available, otherwise no translation:</B> Codon
translation is performed only for those species where the region is
annotated as protein coding.
<LI><B>Use reading frames for species if available, otherwise use default species:</B>
Codon translation is done on those species that are annotated as being protein
coding over the aligned region using species-specific annotation; the remaining
species are translated using the default species annotation. 
</UL></P>
<P>
Codon translation uses the following gene tracks as the basis for
translation, depending on the species chosen (<B>Table 2</B>).  
Species listed in the row labeled &quot;None&quot; do not have 
species-specific reading frames for gene translation.

<BLOCKQUOTE>
<TABLE class="descTbl">
<TR ALIGN=left><TD><B>Gene Track</B></TD><TD><B>Species</B></TD></TR>
<TR ALIGN=left><TD>SGD Genes</TD><TD>S. cerevisae</TD></TR>
<TR ALIGN=left><TD>No annotation</TD><TD>all the other yeast strains</TD></TR>
</TABLE>
<B>Table 2.</B> <EM>Gene tracks used for codon translation.</EM>
</BLOCKQUOTE></P>

<H2>Methods</H2>
<P> 
Best-in-genome pairwise alignments were generated for each species 
using lastz, followed by chaining and netting.  The pairwise alignments
were then multiply aligned using multiz, and
the resulting multiple alignments were assigned 
conservation scores by phastCons.</P>
<P>
The phastCons program computes conservation scores based on a phylo-HMM, a
type of probabilistic model that describes both the process of DNA
substitution at each site in a genome and the way this process changes from
one site to the next (Felsenstein and Churchill 1996, Yang 1995, Siepel and
Haussler 2005).  PhastCons uses a two-state phylo-HMM, with a state for
conserved regions and a state for non-conserved regions.  The value plotted
at each site is the posterior probability that the corresponding alignment
column was "generated" by the conserved state of the phylo-HMM.  These
scores reflect the phylogeny (including branch lengths) of the species in
question, a continuous-time Markov model of the nucleotide substitution
process, and a tendency for conservation levels to be autocorrelated along
the genome (i.e., to be similar at adjacent sites).  The general reversible
(REV) substitution model was used.  Note that, unlike many
conservation-scoring programs, phastCons does not rely on a sliding window
of fixed size, so short highly-conserved regions and long moderately
conserved regions can both obtain high scores.  More information about
phastCons can be found in Siepel <EM>et al</EM>. (2005).</P> 
<P> 
PhastCons currently treats alignment gaps as missing data, which
sometimes has the effect of producing undesirably high conservation scores
in gappy regions of the alignment.  We are looking at several possible ways
of improving the handling of alignment gaps.</P>

<H2>Credits</H2>
<P>
This track was created at UCSC using the following programs:
<UL>
<LI>
Lastz (formerly Blastz) and multiz by Minmei Hou, Scott Schwartz and Webb Miller of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. 
<LI>
AxtBest, axtChain, chainNet, netSyntenic, and netClass 
by Jim Kent at UCSC. 
<LI> PhastCons by Adam Siepel at Cornell University. 
<LI>"Wiggle track" plotting software by Hiram Clawson at UCSC.
</UL>
</P>

<P>The phylogenetic tree is based on the
<A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/yeast_phylogeny.html"
TARGET=_blank><em>Saccharomyces</em> Phylogeny</A> page from the Department
of Genetics at Washington University in St. Louis.

<H2>References</H2>

<H3>Phylo-HMMs and phastCons:</H3>
<P>
Felsenstein J, Churchill GA.
<A HREF="https://academic.oup.com/mbe/article/13/1/93/1055515"
TARGET=_blank>A Hidden Markov Model approach to
variation among sites in rate of evolution</A>.
<em>Mol Biol Evol</em>. 1996 Jan;13(1):93-104.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/8583911" target="_blank">8583911</a>
</p>

<P>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K,
Clawson H, Spieth J, Hillier LW, Richards S, <EM>et al.</EM>
<A HREF="https://genome.cshlp.org/content/15/8/1034"
TARGET=_blank>Evolutionarily conserved elements in vertebrate, insect, worm,
and yeast genomes</A>.
<em>Genome Res</em>. 2005 Aug;15(8):1034-50.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16024819" target="_blank">16024819</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182216/" target="_blank">PMC1182216</a>
</p>

<P>
Siepel A, Haussler D.
<A HREF="http://compgen.cshl.edu/~acs/phylohmm.pdf"
TARGET=_blank>Phylogenetic Hidden Markov Models</A>.
In: Nielsen R, editor. Statistical Methods in Molecular Evolution.
New York: Springer; 2005. pp. 325-351.
</P>

<P>
Yang Z.
<A HREF="http://www.genetics.org/content/139/2/993"
TARGET=_blank>A space-time process model for the evolution of DNA
sequences</A>.
<em>Genetics</em>. 1995 Feb;139(2):993-1005.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/7713447" target="_blank">7713447</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1206396/" target="_blank">PMC1206396</a>
</p>

<H3>Chain/Net:</H3>
<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="http://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<H3>Multiz:</H3>
<P>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM,
Baertsch R, Rosenbloom K, Clawson H, Green ED, <EM>et al.</EM>
<A HREF="https://genome.cshlp.org/content/14/4/708.abstract"
TARGET=_blank>Aligning multiple genomic sequences with the threaded blockset aligner</A>.
<em>Genome Res</em>. 2004 Apr;14(4):708-15.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15060014" target="_blank">15060014</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC383317/" target="_blank">PMC383317</a>
</p>

<H3>Lastz (formerly Blastz):</H3>
<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<p>
Harris RS.
<a href="http://www.bx.psu.edu/~rsharris/rsharris_phd_thesis_2007.pdf"
target="_blank">Improved pairwise alignment of genomic DNA</a>.
<em>Ph.D. Thesis</em>. Pennsylvania State University, USA. 2007.
</p>

