<H2>Description</H2>
<P>
This track shows the alignments of 32 strains of yeast to the $organism
sequence.  The sequence for 26 strains of yeast were obtained from
<A HREF="http://downloads.yeastgenome.org/sequence/strains/"
TARGET=_blank>downloads.yeastgenome.org/sequence/strains/</A>.
An additional six strains were obtained a variety of sequencing centers.
Links to all sequences are listed in the following table:
</P>
<P>
<UL>
<LI><em>S. paradoxus</em> - <A HREF="http://www.broad.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Spar_contigs.fasta" TARGET=_blank>http://www.broad.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Spar_contigs.fasta</A>
<LI><em>S. mikatae</em> - <A HREF="http://www-genome.wi.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Smik_contigs.fasta" TARGET=_blank>http://www-genome.wi.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Smik_contigs.fasta</A>
<LI><em>S. kudriavzevii</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM6553.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM6553.fsa.gz</A>
<LI><em>S. bayanus</em> - <A HREF="http://www-genome.wi.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Sbay_contigs.fasta" TARGET=_blank>http://www-genome.wi.mit.edu/ftp/pub/annotation/fungi/comp_yeasts/S1a.Assembly/Sbay_contigs.fasta</A>
<LI><em>S. castelli</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM476.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM476.fsa.gz</A>
<LI><em>S. kluyveri</em> - <A HREF="http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM479.fsa.gz" TARGET=_blank>http://www.genetics.wustl.edu/saccharomycesgenomes/Contigs/YM479.fsa.gz</A>
<LI><em>AWRI1631</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/AWRI1631/AWRI1631_AWRI_2008_ABSV01000000/AWRI1631.README"
TARGET=_blank>AWRI1631_AWRI_2008_ABSV01000000</A>
<LI><em>AWRI796</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/AWRI796/AWRI796_AWRI_2010_ADVS01000000/AWRI796.README"
TARGET=_blank>AWRI796_AWRI_2010_ADVS01000000</A>
<LI><em>CBS7960</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/CBS7960/CBS7960_WashU_2011_AEWL01000000/CBS7960.README"
TARGET=_blank>CBS7960_WashU_2011_AEWL01000000</A>
<LI><em>CLIB215</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/CLIB215/CLIB215_WashU_2010_AEWP01000000/CLIB215.README"
TARGET=_blank>CLIB215_WashU_2010_AEWP01000000</A>
<LI><em>CLIB324</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/CLIB324/CLIB324_WashU_2011_AEWM01000000/CLIB324.README"
TARGET=_blank>CLIB324_WashU_2011_AEWM01000000</A>
<LI><em>CLIB382</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/CLIB382/CLIB382_WashU_2011_AFDG01000000/CLIB382.README"
TARGET=_blank>CLIB382_WashU_2011_AFDG01000000</A>
<LI><em>EC1118</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/EC1118/EC1118_Genoscope_2009_PRJEA37863/EC1118.README"
TARGET=_blank>EC1118_Genoscope_2009_PRJEA37863</A>
<LI><em>FL100</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/FL100/FL100_WashU_2011_AEWO01000000/FL100.README"
TARGET=_blank>FL100_WashU_2011_AEWO01000000</A>
<LI><em>FostersB</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/FostersB/FostersB_AWRI_2010_AEHH01000000/FostersB.README"
TARGET=_blank>FostersB_AWRI_2010_AEHH01000000</A>
<LI><em>FostersO</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/FostersO/FostersO_AWRI_2010_AEEZ01000000/FostersO.README"
TARGET=_blank>FostersO_AWRI_2010_AEEZ01000000</A>
<LI><em>JAY291</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/JAY291/JAY291_Duke_2009_ACFL01000000/JAY291.README"
TARGET=_blank>JAY291_Duke_2009_ACFL01000000</A>
<LI><em>LalvinQA23</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/LalvinQA23/LalvinQA23_AWRI_2011_ADVV01000000/LalvinQA23.README"
TARGET=_blank>LalvinQA23_AWRI_2011_ADVV01000000</A>
<LI><em>M22</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/M22/M22_WashU_2008_ABPC01000000/M22.README"
TARGET=_blank>M22_WashU_2008_ABPC01000000</A>
<LI><em>PW5</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/PW5/PW5_WashU_2011_AFDC01000000/PW5.README"
TARGET=_blank>PW5_WashU_2011_AFDC01000000</A>
<LI><em>RM11-1a</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/RM11-1a/RM11-1a_Broad_2005_AAEG01000000/RM11-1a.README"
TARGET=_blank>RM11-1a_Broad_2005_AAEG01000000</A>
<LI><em>Sigma1278b</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/Sigma1278b/Sigma1278b_MIT_2009_ACVY01000000/Sigma1278b.README"
TARGET=_blank>Sigma1278b_MIT_2009_ACVY01000000</A>
<LI><em>T7</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/T7/T7_WashU_2011_AFDE01000000/T7.README"
TARGET=_blank>T7_WashU_2011_AFDE01000000</A>
<LI><em>T73</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/T73/T73_WashU_2011_AFDF01000000/T73.README"
TARGET=_blank>T73_WashU_2011_AFDF01000000</A>
<LI><em>UC5</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/UC5/UC5_WashU_2011_AFDD01000000/UC5.README"
TARGET=_blank>UC5_WashU_2011_AFDD01000000</A>
<LI><em>Vin13</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/Vin13/Vin13_AWRI_2010_ADXC01000000/Vin13.README"
TARGET=_blank>Vin13_AWRI_2010_ADXC01000000</A>
<LI><em>VL3</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/VL3/VL3_AWRI_2011_AEJS01000000/VL3.README"
TARGET=_blank>VL3_AWRI_2011_AEJS01000000</A>
<LI><em>W303</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/W303/W303_MPG_2011/W303.README"
TARGET=_blank>W303_MPG_2011</A>
<LI><em>Y10</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/Y10/Y10_WashU_2011_AEWK01000000/Y10.README"
TARGET=_blank>Y10_WashU_2011_AEWK01000000</A>
<LI><em>YJM269</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/YJM269/YJM269_WashU_2011_AEWN01000000/YJM269.README"
TARGET=_blank>YJM269_WashU_2011_AEWN0100000</A>
<LI><em>YJM789</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/YJM789/YJM789_Stanford_2007_AAFW02000000/YJM789.README"
TARGET=_blank>YJM789_Stanford_2007_AAFW02000000</A>
<LI><em>YPS163</em> - <A HREF="http://downloads.yeastgenome.org/sequence/strains/YPS163/YPS163_WashU_2008_ABPD01000000/YPS163.README"
TARGET=_blank>YPS163_WashU_2008_ABPD01000000</A>
</UL>
</P>

<H2>Methods</H2>
<P>
Each genome was aligned to the $organism sequence with the
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>lastz</A> aligner
from the Penn State Bioinformatics Group.  Default parameters for
<em>lastz</em> were used.  The resulting <em>lav</em> outputs were
converted to <em>psl</em> format with the kent source tree program
<em>lavToPsl</em>.  The <em>psl</em> alignments were chained together with
the kent source tree program <em>axtChain</em> with options minimum
score of 1000 and the <em>medium</em> linear gap scoring matrix.
The <em>chain</em> results were converted to <em>psl</em> format
to load into the UCSC genome database.
</P>

<H2>Credits</H2>
<P> This track was created using the following programs:
<UL>
<LI> Alignment tools: lastz by Minmei Hou, Scott Schwartz and Webb
Miller of the <A HREF="http://www.bx.psu.edu/miller_lab/" 
TARGET=_blank>Penn State Bioinformatics Group</A>
<LI> Chaining:  axtChain by Jim Kent at UCSC
</UL>
</P>

<P> The data and presentation of these alignments were prepared by
<A HREF="mailto:&#104;&#105;&#114;a&#109;&#64;&#115;&#111;&#101;
.&#117;&#99;&#115;&#99;.&#101;&#100;u">Hiram Clawson</A>.
</P>


<H2>References</H2>

<H3>Chain/Net:</H3>
<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, Haussler D.
<A HREF="http://www.pnas.org/cgi/content/abstract/1932072100v1"
TARGET=_blank>Evolution's cauldron:
duplication, deletion, and rearrangement in the mouse and human genomes</A>.
<EM>Proc Natl Acad Sci U S A</EM>. 2003 Sep 30;100(20):11484-9.
</P>

<H3>Blastz:</H3>
<P>
Chiaromonte F, Yap VB, Miller W.
<A HREF="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11928468&dopt=Abstract"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>.
<EM>Pac Symp Biocomput</EM>. 2002;:115-26.
</P>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison RC,
Haussler D, Miller W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<EM>Genome Res</EM>. 2003 Jan;13(1):103-7.
</P>

