<html>
<title>UW protein binding footprints</title>
<body>

<h1>Description</h1>

<p>
The orchestrated binding of transcriptional activators and repressors
to specific DNA sequences in the context of chromatin defines the
regulatory program of eukaryotic genomes.  We developed a digital
approach to assay regulatory protein occupancy on genomic DNA in vivo
by dense mapping of individual DNase I cleavages from intact nuclei
using massively parallel DNA sequencing.  Analysis of &gt;23 million
cleavages across the <i>Saccharomyces cerevisiae</i> genome revealed
thousands of protected regulatory protein footprints, enabling <i>de
novo</i> derivation of factor binding motifs as well as the
identification of hundreds of novel binding sites for major
regulators.  We observed striking correspondence between
nucleotide-level DNase I cleavage patterns and protein-DNA
interactions determined by crystallography.  The data also yielded a
detailed view of larger chromatin features including positioned
nucleosomes flanking factor binding regions.  Digital genomic
footprinting provides a powerful approach to delineate the
cis-regulatory framework of any organism with an available genome
sequence. </p>

</ul>

<h1>Display Conventions and Configuration</h1>

<p>
DNaseI-seq cleavage counts are displayed at nucleotide resolution,
along with a 'mappability' track that indicates whether tag sequences
starting at that location on both the forward and the reverse strands can be
uniquely mapped to the yeast genome.  Finally, the set of footprints
with q&nbsp;values &lt;0.1 are included, where the q&nbsp;value is
defined as the minimal false discovery rate threshold at which the
given footprint is deemed significant.  The name associated with each
footprint is its q&nbsp;value.</p>

<h1>Methods</h1>

<p>
To visualize regulatory protein occupancy across the genome of
<i>Saccharomyces cerevisiae</i>, DNase I digestion of yeast nuclei was
coupled with massively parallel DNA sequencing to create a dense
whole-genome map of DNA template accessibility at the 
nucleotide-level.</p>

<p>
Yeast nuclei were isolated and treated with a DNase I concentration
sufficient to release short (&lt;300 bp) DNA fragments.  Small
fragments were derived from two DNase I "hits" in close proximity.
Each end of those fragments represents an in vivo DNase I cleavage
site. The sequence and hence genomic location of these sites were then
determined by DNA sequencing.</p>

<p>
Footprints were identified using a computational algorithm that
evaluates short regions (between 8 and 30 bp) over which the DNase I
cleavage density was significantly reduced compared with the
immediately flanking regions.  FDR thresholds were assigned to each
footprint by comparing p-values obtained from real and shuffled
cleavage data.</p>

<p>
Detailed methods are given in Hesselberth <i>et al.</i> (2009), and
supplementary data and source code are available
<a href="https://noble.gs.washington.edu/proj/footprinting/" target="_blank">here</a>.</p>

<h1>Credits</h1>

<p>
This track was produced at the University of Washington by Jay
  R. Hesselberth, Xiaoyu Chen, Zhihong Zhang, Peter J. Sabo, Richard
  Sandstrom, Alex P. Reynolds, Robert E. Thurman, Shane Neph, Michael
  S. Kuehn, William S. Noble (william-noble@u.washington.edu), Stanley
  Fields (fields@u.washington.edu) and John A. Stamatoyannopoulos
  (jstam@stamlab.org).</p>

<h1>References</h1>

<p>
Hesselberth JR, Chen X, Zhang Z, Sabo PJ, Sandstrom R, Reynolds AP, Thurman RE, Neph S, Kuehn MS,
Noble WS <em>et al</em>.
<a href="https://www.nature.com/articles/nmeth.1313"
target="_blank"> Global mapping of protein-DNA interactions <i>in vivo</i> by digital genomic footprinting</a>.
<em>Nat Methods</em>. 2009 Apr;6(4):283-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19305407" target="_blank">19305407</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2668528/" target="_blank">PMC2668528</a>
</p>
</body>
</html>
