<H2>Description</H2>
<P>
This track shows alignments between $organism expressed sequence tags 
(ESTs) in <A HREF="https://www.ncbi.nlm.nih.gov/Genbank/index.html" 
TARGET=_blank>GenBank</A> and the genome. ESTs are single-read sequences, 
typically about 500 bases in length, that usually represent fragments of 
transcribed genes.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay" 
TARGET=_blank>PSL alignment tracks</A>. In dense display mode, darker shading
indicates a larger number of aligned ESTs. </P>
<P>
The strand information (+/-) indicates the
direction of the match between the EST and the matching
genomic sequence.  It bears no relationship to the direction
of transcription of the RNA with which it might be associated.</P>
<P>
The description page for this track has a filter that can be used to change 
the display mode, alter the color, and include/exclude a subset of items 
within the track. This may be helpful when many items are shown in the track 
display, especially when only some are relevant to the current task.  </P>
<P>
To use the filter:
<OL>
<LI>Type a term in one or more of the text boxes to filter the EST
display. For example, to apply the filter to all ESTs expressed in a specific
organ, type the name of the organ in the tissue box. To view the list of 
valid terms for each text box, consult the table in the Table Browser that 
corresponds to the factor on which you wish to filter. For example, the 
&quot;tissue&quot; table contains all the types of tissues that can be 
entered into the tissue text box. Multiple terms may be entered at once, 
separated by a space. Wildcards may also be used in the
filter.
<LI>If filtering on more than one value, choose the desired combination
logic. If &quot;and&quot; is selected, only ESTs that match all filter 
criteria will be highlighted. If &quot;or&quot; is selected, ESTs that 
match any one of the filter criteria will be highlighted.
<LI>Choose the color or display characteristic that should be used to 
highlight or include/exclude the filtered items. If &quot;exclude&quot; is 
chosen, the browser will not display ESTs that match the filter criteria. 
If &quot;include&quot; is selected, the browser will display only those 
ESTs that match the filter criteria.
</OL></P>
<P>
This track may also be configured to display base labeling, a feature that
allows the user to display all bases in the aligning sequence or only those 
that differ from the genomic sequence. For more information about this option,
click 
<A HREF="../goldenPath/help/hgBaseLabel.html" TARGET=_blank>here</A>.
Several types of alignment gap may also be colored; 
for more information, click 
<A HREF="../goldenPath/help/hgIndelDisplay.html" TARGET=_blank>here</A>.
</P>

<H2>Methods</H2>
<P>
To make an EST, RNA is isolated from cells and reverse
transcribed into cDNA.  Typically, the cDNA is cloned
into a plasmid vector and a read is taken from the 5'
and/or 3' primer.  For most &mdash; but not all &mdash; ESTs, the
reverse transcription is primed by an oligo-dT, which
hybridizes with the poly-A tail of mature mRNA. The
reverse transcriptase may or may not make it to the 5'
end of the mRNA,  which may or may not be degraded.</P>
<P>
In general, the 3' ESTs mark the end of transcription
reasonably well, but the 5' ESTs may end at any point
within the transcript.  Some of the newer cap-selected
libraries cover transcription start reasonably well. Before the 
cap-selection techniques
emerged, some projects used random rather than poly-A
priming in an attempt to retrieve sequence distant from the
3' end.  These projects were successful at this, but as
a side effect also deposited sequences from unprocessed
mRNA and perhaps even genomic sequences into the EST databases.
Even outside of the random-primed projects, there is a
degree of non-mRNA contamination. Because of this, a
single unspliced EST should be viewed with considerable
skepticism.  </P>
<P>
To generate this track, $organism ESTs from GenBank were aligned 
against the genome using blat. Note that the maximum intron length
allowed by blat is 750,000 bases, which may eliminate some ESTs with very 
long introns that might otherwise align. When a single 
EST aligned in multiple places, the alignment having the 
highest base identity was identified.  Only alignments having
a base identity level within 0.5% of the best and at least 96% base identity 
with the genomic sequence are displayed in this track.</P>

<H2>Credits</H2>
<P>
This track was produced at UCSC from EST sequence data
submitted to the international public sequence databases by 
scientists worldwide.</P>

<H2>References</H2>
<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, 
Wheeler DL. 
<A HREF="http://nar.oxfordjournals.org/content/32/suppl_1/D23.abstract" 
TARGET=_blank>GenBank: update</A>. <em>Nucleic Acids Res.</em>
2004 Jan 1;32(Database issue):D23-6.</P>
<P>
Kent WJ.
<A HREF="http://genome.cshlp.org/content/12/4/656.full"
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.</P>
