<H2>Description</H2>
<P>
The mRNA track shows alignments between $organism mRNAs
in <A HREF="https://www.ncbi.nlm.nih.gov/Genbank/index.html" 
TARGET=_blank>GenBank</A> and the genome. </P>

<H2>Display Conventions and Configuration</H2>
<P>
This track follows the display conventions for 
<A HREF="../goldenPath/help/hgTracksHelp.html#PSLDisplay" 
TARGET=_blank>PSL alignment tracks</A>. In dense display mode, the items that
are more darkly shaded indicate matches of better quality. </P>
<P>
The description page for this track has a filter that can be used to change 
the display mode, alter the color, and include/exclude a subset of items 
within the track. This may be helpful when many items are shown in the track 
display, especially when only some are relevant to the current task.  </P>
<P>
To use the filter:
<OL>
<LI>Type a term in one or more of the text boxes to filter the mRNA 
display. For example, to apply the filter to all mRNAs expressed in the 
liver, type &quot;liver&quot; in the tissue box. To view the list of 
valid terms for each text box, consult the table in the Table Browser that 
corresponds to the factor on which you wish to filter. For example, the 
&quot;tissue&quot; table contains all the types of tissues that can be 
entered into the tissue text box. Wildcards may also be used in the
filter.
<LI>If filtering on more than one value, choose the desired combination
logic. If &quot;and&quot; is selected, only mRNAs that match all filter 
criteria will be highlighted. If &quot;or&quot; is selected, mRNAs that 
match any one of the filter criteria will be highlighted.
<LI>Choose the color or display characteristic that should be used to 
highlight or include/exclude the filtered items. If &quot;exclude&quot; is 
chosen, the browser will not display mRNAs that match the filter criteria. 
If &quot;include&quot; is selected, the browser will display only those 
mRNAs that match the filter criteria.
</OL></P>
<P>
This track may also be configured to display codon coloring, a feature that
allows the user to quickly compare mRNAs against the genomic sequence. For more 
information about this option, click 
<A HREF="../goldenPath/help/hgCodonColoringMrna.html" TARGET=_blank>here</A>.
Several types of alignment gap may also be colored; 
for more information, click 
<A HREF="../goldenPath/help/hgIndelDisplay.html" TARGET=_blank>here</A>.
</P>

<H2>Methods</H2>
<P>
GenBank $organism mRNAs were aligned against the genome using the 
blat program. When a single mRNA aligned in multiple places, 
the alignment having the highest base identity was found. 
Only alignments having a base identity level within 0.5% of
the best and at least 96% base identity with the genomic sequence were kept.
</P>

<H2>Credits</H2>
<P>
This track was produced at UCSC from mRNA sequence data
submitted to the international public sequence databases by 
scientists worldwide.</P>

<H2>References</H2>
<P>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, 
Wheeler DL. 
<A HREF="http://nar.oxfordjournals.org/content/32/suppl_1/D23.abstract" 
TARGET=_blank>GenBank: update</A>. <em>Nucleic Acids Res.</em>
2004 Jan 1;32(Database issue):D23-6.</P>
<P>
Kent WJ.
<A HREF="http://genome.cshlp.org/content/12/4/656.full"
TARGET=_blank>BLAT - the BLAST-like alignment tool</A>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.</P>
