<H2>Description</H2>
<P>
This track shows blastz alignments of the $o_organism assembly 
($o_db, $o_date) to the $organism genome, filtered to display only the 
best alignment for any given region of the $organism genome. The track has 
an optional feature that color codes alignments to indicate the chromosomes
from which they are derived in the aligning assembly. To activate the color
feature, click the <em>on</em> radio button next to &quot;Color track 
based on chromosome&quot; on the track description page.</P>

<H2>Methods</H2>
<P>
For blastz, 12 of 19 seeds were used and then scored using: 
<PRE>
      A     C     G     T
A    91  -114   -31  -123
C  -114   100  -125   -31
G   -31  -125   100  -114
T  -123   -31  -114    91

O = 400, E = 30, K = 3000, L = 3000, M = 50
</PRE>
</P>
<P>
A second pass was made at reduced stringency (7mer seeds and
MSP threshold of K=2200) to attempt to fill in gaps of up to about 10K bp.
Lineage-specific repeats were abridged during this alignment.</P>

<H2>Using the Filter</H2>
<P>
This track has a filter that can be used to change the display mode, 
turn on the chromosome color track, or filter the display output by
chromosome. The filter is located at the top of the track description page,
which is accessed via the small button to the left of the track's graphical
display or through the link on the track's control menu. 
<UL>
<LI><B>Color track</B>: To display the chromosome color track, click the 
<em>on</em> button next to &quot;Color track based on chromosome&quot;. 
When the color track is activated, each of the items within the annotation 
track will be colored to show the chromosome in the aligning genome 
assembly from which the alignment originated.
<LI><B>Chromosome filter</B>: To display only alignments from a specific 
chromomsome in the aligning assembly, type the chromosome number (in the 
form <I>chrN</I>) in the text box to the right of &quot;Filter by 
chromosome&quot;. For example, to display alignments from chromosome 6, 
type &quot;chr6&quot;.
</UL><P>
When you have finished configuring the filter, click the <I>Submit</I> 
button.</P>

<H2>Credits</H2>
<P>
These alignments were contributed by Scott Schwartz of the 
<A HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics 
Group</A>. The best-in-genome filtering was done using UCSC's 
<A HREF="http://www.genome.org/cgi/content/full/13/1/103" 
TARGET=_blank>axtBest</A> program. Mouse sequence data were provided by the
Mouse Genome Sequencing Consortium. </P>

<H2>References</H2>
<P>
Schwartz, S., Kent, W.J., Smit, A., Zhang, Z., Baertsch, R., Hardison, R.,
Haussler, D., and Miller, W.
<A HREF="http://www.genome.org/cgi/content/abstract/13/1/103"
TARGET=_blank>Human-mouse alignments with BLASTZ</A>.
<em>Genome Res.</em> <B>13</B>(1), 103-107 (2003).</P>
