
<h3>Description</h3>

<p>The data are shown in a tabular format in which each column of
colored boxes represents the variation in genomic DNA levels from a 
normal cell line for a
given clone across all of the NCI60 cell lines, and each row
represents the measured genomic DNA levels for clones in a single
sample. The variation in genomic DNA levels for each clone is
represented by a color scale, in which green indicates an increase in
genomic DNA levels, and red indicates a decrease in genomic DNA
levels, relative to the reference sample. The saturation of the color
corresponds to the magnitude of transcript variation. A black color
indicates an undetectable change in genomic DNA, while a gray box
indicates missing data.

<h3>Display Options</h3>
This track has options to customize tissue types presented and
the color of the display.

<br><b>Cell Line: </b> This option is only valid when the track is 
displayed in full. It determines how the experiments are displayed. The
options are:
<ul>
<li><b>Tissue Averages: </b> Displays the average of the log ratio scores of all cell lines 
from the different tissue types.</li>
<li><b>All Cell Lines: </b> Displays the log ratio score for all cell line experiments.
</li>
<li><b>Specific Tissues: </b> Displays the log ratio score for all cell lines belonging
to a given tissue type.</li>
</ul>
<b>Color Scheme: </b> 
Data are presented using two color false display. By default
the colors of green -> positive log ratio, red -> negative log ratio are
used.
However, blue can be substituted for green for those who are color blind.

<h3>Details Page</h3> 
On the details page the probes presented
correspond to those contained in window range seen on the Genome
Browser, the exon probe and experiment selected are highlighted in
blue.
