<H2>Description</H2>
<P>
This track shows protein-coding gene predictions generated by 
<a href=http://contra.stanford.edu/contrast/ target="_blank">CONTRAST</a>.  
Each predicted exon is colored according to confidence level: green (high
confidence), orange (medium confidence), or red (low confidence).
</P>

<H2>Methods</H2>
<P>
CONTRAST predicts protein-coding genes from a multiple genomic
alignment using a combination of discriminative machine learning techniques.  A two-stage approach is used, in which output from local classifiers is combined with a global model of gene structure.  CONTRAST is trained using a novel procedure designed to
maximize expected coding region boundary detection accuracy.

<P>
Please see the 
<a href=http://contra.stanford.edu/contrast/ target="_blank">CONTRAST web site</a> for details on how these predictions were generated and an estimate of accuracy. 

<H2>Credits</H2>
<P>
Thanks to Samuel Gross of the <a href="http://www.serafimb.org/people.html" target="_blank">
Batzoglou lab</a> at Stanford University for providing these predictions.

<H2>References</H2>
<p>
Gross SS, Do CB, Sirota M, Batzoglou S.
<a href="https://genomebiology.biomedcentral.com/articles/10.1186/gb-2007-8-12-r269"
target="_blank">CONTRAST: a discriminative, phylogeny-free approach to multiple informant de novo
gene prediction</a>.
<em>Genome Biol</em>. 2007;8(12):R269.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18096039" target="_blank">18096039</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2246271/" target="_blank">PMC2246271</a>
</p>
