<h2>Description</h2>
<p>
The Pathways and Gene Interactions track shows a summary of gene interaction and pathway data
collected from two sources: curated pathway/protein-interaction databases and interactions found
through text mining of PubMed abstracts.</p>

<h2>Display Conventions and Configuration</h2>
<h3>Track Display</h3>
<p>
The track features a single item for each gene loci in the genome. On the item itself, the gene
symbol for the loci is displayed followed by the top gene interactions noted by their gene symbol.
Clicking an item will take you a
<a href="../../goldenPath/help/hgGeneGraph.html" target="_blank">gene interaction graph</a>
that includes detailed information on the support for the various interactions.</p>

<p>
Items are colored based on the number of documents supporting the interactions of a
particular gene. Genes with &gt;100 supporting documents are colored
<strong><font color="#000000">black</font></strong>, genes with &gt;10 but &lt;100
supporting documents are colored <strong><font color="#000080">dark blue</font></strong>, and
those with &gt;10 supporting documents are colored
<strong><font color="#add8e6">light blue</font></strong>.</p>

<h3>Pathway and Gene Interaction Display</h3>
<p>
See the
<a href="../../goldenPath/help/hgGeneGraph.html#configure" target="_blank">help documentation</a>
accompanying this gene interaction graph for more information on its configuration.</p>

<h2>Methods</h2>
<p>
The pathways and gene interactions were imported from a number of databases and mined from
millions of PubMed abstracts. More information can be found in the
&quot;<a href="../../goldenPath/help/hgGeneGraph.html#methods" target="_blank">Data Sources
and Methods</a>&quot;
section of the help page for the gene interaction graph.</p>

<h2>Data Access</h2>
<p>
The underlying data for this track can be accessed interactively through the
<a href="hgTables" target="_blank">Table Browser</a> or
<a href="hgIntegrator" target="_blank">Data Integrator</a>.
For automated analysis, the genome annotation is stored in a bigBed file that can be obtained
from the
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/" target="_blank">download server</a>.
The data for this track is spread across a number of files and tables. The data underlying the
graphical display is in <a href="../../goldenPath/help/bigBed.html" target="_blank">bigBed</a>
formatted file named <tt>interactions.bb</tt>. Individual regions or the whole genome annotation
can be obtained using our tool <tt>bigBedToBed</tt>. Instructions
for downloading source code and precompiled binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads"
target="_blank">here</a>. The tool can also
be used to obtain only features within a given range, for example:
<p>
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/interactions.bb
-chrom=chr6 -start=0 -end=1000000 stdout</tt>
</p>

<h2>Credits</h2>
<p>
The text-mined data for the gene interactions and pathways were generated by Chris Quirk and
Hoifung Poon as part of
<a href="https://hanover.azurewebsites.net" target="_blank">Microsoft Research, Project
Hanover</a>.</p>

<p>
Pathway data was provided by the databases listed under
&quot;<a href="../../goldenPath/help/hgGeneGraph.html#methods" target="_blank">Data Sources
and Methods</a>&quot;
section of the help page for the gene interaction graph.
In particular, thank you to Ian Donaldson from IRef for his
unique collection of interaction databases.</p>

<p>
The short gene descriptions are a merge of the <a href="http://hprd.org" target="_blank">HPRD</a>
and <a href="http://pantherdb.org" target="_blank">PantherDB</a> gene/molecule classifications. Thanks to Arun Patil from
HPRD for making them available as a download.</p>

<p>
The track display and gene interaction graph
were developed at the UCSC Genome Browser by Max Haeussler.</p>

<h2>References</h2>
<p>
Poon H, Quirk C, DeZiel C, Heckerman D.
<a href="https://academic.oup.com/bioinformatics/article/30/19/2840/2422228/Literome-PubMed-scale-genomic-knowledge-base-in"
target="_blank">Literome: PubMed-scale genomic knowledge base in the cloud</a>
<em>Bioinformatics</em>. 2014 Oct;30(19):2840-2.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/24939151" target="_blank">24939151</a></p>
