<H2>Description</H2>
<P>
This track shows Mouse quantitative trait loci (QTLs) from 
<A HREF="http://www.informatics.jax.org/" 
TARGET=_BLANK>Mouse Genome Informatics</A> (MGI) at the 
<A HREF="https://www.jax.org/" TARGET=_BLANK>Jackson Laboratory</A> 
that have been coarsely mapped by UCSC to the $Organism genome using 
stringently filtered cross-species alignments.  
A quantitative trait locus (QTL) is a polymorphic locus that contains alleles
which differentially affect the expression of a continuously distributed 
phenotypic trait. Usually a QTL is a marker described by statistical 
association to quantitative variation in the particular phenotypic trait that
is thought to be controlled by the cumulative action of alleles at multiple 
loci.</P>
<P>
To map the Mouse QTLs to $Organism, UCSC's chained and netted blastz
alignments of Mouse to $Organism were filtered to retain only those with
minimum length of 20,000 bases in both Mouse and $Organism, and minimum 
score of 10,000.  This removed many valid-but-short alignments.  This
choice was made because QTLs in general are extremely large and
approximate regions.  After the alignment filtering, UCSC's liftOver
program was used to map Mouse regions to $Organism via the filtered
alignments.</P>
<P>
For the purpose of cross-species mapping, MGI QTLs were divided into
two categories: QTLs whose genomic coordinates span the entire
confidence interval (often several million bases), and QTLs for which
only the STS marker with the peak score was given, resulting in
genomic coordinates for very small regions (most less than 300 bases).
QTLs in the latter set were so small as to make mapping impossible in many 
cases, so their coordinates were padded by 50,000 bases before and 
after, for a total size of approximately 100,000 bases, a 
conservative proxy for the unknown confidence interval.  The two 
categories of QTL are displayed in subtracks: MGI Mouse QTL for the 
unmodified QTLs and MGI Mouse QTL Padded for the single-marker QTLs 
that were padded to 100,000 bases.
</P>
<P>
To get a sense of how many genomic rearrangments between Mouse and
$Organism are in the region of a particular Mouse QTL, you may want to
view the $Organism Nets track in the Mouse $o_date genome browser.  
In the position/search box, enter the name of the Mouse QTL of interest.
</P>

<H2>Credits</H2>
<P>
Thanks to 
<A HREF="http://www.informatics.jax.org/" TARGET=_BLANK>MGI</A> 
at the Jackson Laboratory, 
and Bob Sinclair in particular, for providing these data.</P>
</P>
