<P>This track displays the mouse reference alignment produced by
Scott Schwartz and 
<A HREF="mailto:&#119;e&#98;&#98;&#64;&#98;&#105;&#111;.&#99;s&#101;.&#112;&#115;&#117;.e&#100;&#117;">Webb Miller</A> 
of the <A HREF="http://www.bx.psu.edu/miller_lab/">Penn State Bioinformatics Group</A>.

<H2>Methods</H2>

<P>The reference alignments are an attempt to reduce the matches between
the human genome sequence (specifically, the Golden Path assembly from the
University of California at Santa Cruz) and the mouse whole-genome shotgun
data to canonical alignments in which each position in the human sequence is
aligned to at most one position in a mouse sequence.  For human chromsome 22,
This set of alignments was computed by aligning all (trimmed and
masked) mouse reads with the human sequence using the blastz program.

<P>The following are the main steps in our current strategy for producing
reference alignments from a collection of (potentially overlapping) blastz
alignments.

<OL>
<LI> Regions that align with a very large number of mouse reads are
   identified.  currently, this means intervals in which each human position
   is aligned to at least 50 mouse reads.  These <EM>highly duplicated</EM>
   segments are excluded from further analysis.

<LI> Alignments are trimmed whenever there exists another alignment with a
   higher percent nucleotide identity in the overlap.  For instance, suppose
   alignment A covers human positions w-z, and B covers x-y, with w < x < y <
   z.  If on interval x-y, B has a higher percent identity than does A, then A
   is broken into an alignment from w to x-1 and an alignment from y+1 to z,
   with the segment from x to y being discarded in favor of B.

<LI> An effort is made to determine which human segments are aligned to more
   than one region of the mouse genome (<EM>moderately duplicated</EM>).
   Whenever alignments are seen to have an appreciable overlap, the two
   portions of mouse reads are compared.  If they have at least 95% nucleotide
   identity, they are treated as coming from the same part of the mouse genome
   (allowing 5% sequencing error between the reads); otherwise the region is
   labeled as duplicated.

<LI> The final processing steps are to identify regions, not annotated as
   exons, that are particularly well conserved (<EM>highly conserved></EM>).
</OL>

<H2>Credits</H2>
<P>This track is produced from mouse sequence data provided by the 
Mouse Genome Sequencing Consortium. 
