<H2>Description</H2> 
<P> 
This track shows predictions of conserved elements produced by the phastCons
program.  PhastCons is part of the <a href="http://compgen.cshl.edu/phast/" target="_blank">
PHAST</a> (PHylogenetic Analysis with 
Space/Time models) package.  The predictions are based on a phylogenetic hidden 
Markov model (phylo-HMM), a type of probabilistic model that describes both 
the process of DNA substitution at each site in a genome and the way this 
process changes from one site to the next. </P>

<H2>Methods</H2>
<P>
Best-in-genome pairwise alignments were generated for
each species using blastz, followed by chaining and netting.  A multiple
alignment was then constructed from these pairwise alignments using multiz.
Predictions of conserved elements were then obtained by running phastCons
on the multiple alignments with the --most-conserved option.</P>
<P>
PhastCons constructs a two-state phylo-HMM with a state for conserved
regions and a state for non-conserved regions.  The two states share a
single phylogenetic model, except that the branch lengths of the tree
associated with the conserved state are multiplied by a constant scaling
factor <em>rho</em> (0 &lt;= <em>rho</em> &lt;= 1).  The free parameters of the
phylo-HMM, including the scaling factor <em>rho</em>, are estimated from
the data by maximum likelihood using an EM algorithm.  This procedure is
subject to certain constraints on the "coverage" of the genome by conserved
elements and the "smoothness" of the conservation scores.  Details can be
found in Siepel <em>et al</em>. (2005).</P>
<P>
The predicted conserved elements are segments of the alignment that are
likely to have been "generated" by the conserved state of the phylo-HMM.
Each element is assigned a log-odds score equal to its log probability
under the conserved model minus its log probability under the non-conserved
model.  The "score" field associated with this track contains transformed
log-odds scores, taking values between 0 and 1000.  (The scores are
transformed using a monotonic function of the form a * log(x) + b.)  The
raw log odds scores are retained in the "name" field and can be seen on the
details page or in the browser when the track's display mode is set to
"pack" or "full".</P>

<H2>Credits</H2>
<P>
This track was created at UCSC using the following programs:
<UL>
	<LI>Blastz and multiz by Minmei Hou, Scott Schwartz and Webb Miller of the <A 
	HREF="http://www.bx.psu.edu/miller_lab/" TARGET=_blank>Penn State Bioinformatics Group</A>. 
	<LI>AxtBest, axtChain, chainNet, netSyntenic, and netClass by Jim Kent at UCSC. 
	<LI>PhastCons by Adam Siepel at Cornell University. 
</UL>
</P>

<H2>References</H2>

<H3>PhastCons</H3>
<P>
Siepel A, Bejerano G, Pedersen JS, Hinrichs AS, Hou M, Rosenbloom K, Clawson H, Spieth J, Hillier
LW, Richards S <em>et al</em>.
<A HREF="https://genome.cshlp.org/content/15/8/1034"
TARGET=_blank>Evolutionarily conserved elements in vertebrate, insect, worm, 
and yeast genomes</A>.
<em>Genome Res</em>. 2005 Aug;15(8):1034-50.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/16024819" target="_blank">16024819</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1182216/" target="_blank">PMC1182216</a>
</p>

<H3>Chain/Net</H3>
<P>
Kent WJ, Baertsch R, Hinrichs A, Miller W, and Haussler D.
<A HREF="http://www.pnas.org/content/100/20/11484"
TARGET=_blank>Evolution's cauldron: Duplication, deletion, and rearrangement
in the mouse and human genomes</A>.
<em>Proc Natl Acad Sci U S A</em>. 2003 Sep 30;100(20):11484-9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14500911" target="_blank">14500911</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC208784/" target="_blank">PMC208784</a>
</p>

<H3>Multiz</H3>
<P>
Blanchette M, Kent WJ, Riemer C, Elnitski L, Smit AF, Roskin KM, Baertsch R, Rosenbloom K, Clawson
H, Green ED <em>et al</em>.
<A HREF="https://genome.cshlp.org/content/14/4/708.abstract"
TARGET=_blank>Aligning multiple genomic sequences with the threaded blockset aligner</A>.
<em>Genome Res</em>. 2004 Apr;14(4):708-15.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15060014" target="_blank">15060014</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC383317/" target="_blank">PMC383317</a>
</p>

<H3>Blastz</H3>
<P>
Chiaromonte F, Yap VB, Miller W. 
<A HREF="http://psb.stanford.edu/psb-online/proceedings/psb02/chiaromonte.pdf"
TARGET=_blank>Scoring pairwise genomic sequence alignments</A>. 
<em>Pac Symp Biocomput</em>. 2002:115-26.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11928468" target="_blank">11928468</a>
</p>

<P>
Schwartz S, Kent WJ, Smit A, Zhang Z, Baertsch R, Hardison R, 
Haussler D, and Miller W.
<A HREF="https://genome.cshlp.org/content/13/1/103.abstract"
TARGET=_blank>Human-Mouse Alignments with BLASTZ</A>. 
<em>Genome Res</em>. 2003 Jan;13(1):103-7.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/12529312" target="_blank">12529312</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC430961/" target="_blank">PMC430961</a>
</p>
