<h2>Description</h2>

<p>
The RefSeq Genes track shows known $organism protein-coding and
non-protein-coding genes taken from the NCBI RNA reference sequences
collection (RefSeq). The data underlying this track are updated weekly.</p>

<p>
Please visit the <a href="https://www.ncbi.nlm.nih.gov/projects/RefSeq/update.cgi" 
target="_blank">Feedback for Gene and Reference Sequences (RefSeq)</a> page to
make suggestions, submit additions and corrections, or ask for help concerning
RefSeq records.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
This track follows the display conventions for
<a href="../goldenPath/help/hgTracksHelp.html#GeneDisplay" target="_blank">
gene prediction tracks</a>.
The color shading indicates the level of review the RefSeq record has
undergone: predicted (light), provisional (medium), reviewed (dark).
</p>

<p>
The item labels and display colors of features within this track can be
configured through the controls at the top of the track description page.
<ul>
<li><b>Label:</b> By default, items are labeled by gene name. Click the
appropriate Label option to display the accession name instead of the gene
name, show both the gene and accession names, or turn off the label
completely.</li>
<li><b>Codon coloring:</b> This track contains an optional codon coloring
feature that allows users to quickly validate and compare gene predictions.
To display codon colors, select the <em>genomic codons</em> option from the
<em>Color track by codons</em> pull-down menu. For more information about this
feature, go to the
<a href="../goldenPath/help/hgCodonColoring.html" target="_blank">
Coloring Gene Predictions and Annotations by Codon</a> page.</li>
<li><b>Hide non-coding genes:</b> By default, both the protein-coding and
non-protein-coding genes are displayed.  If you wish to see only the coding
genes, click this box.</li>
</ul>
</p>

<h2>Methods</h2>

<p>
RefSeq RNAs over 20 base pairs were aligned against the $organism genome using BLAT.
Those alignments with less than 16 bases aligning outside of annotated repeats were discarded.
Next, the alignment with the highest base identity between the RNA and the genomic sequence was chosen.
For each RNA, alignments not within 0.1% of the base identity of the best alignment were discarded.
</p>

<h2>Credits</h2>

<p>
This track was produced at UCSC from RNA sequence data generated by scientists
worldwide and curated by the NCBI
<a href="https://www.ncbi.nlm.nih.gov/refseq/" target="_blank">RefSeq project</a>.
</p>

<h2>References</h2>

<p>
Kent WJ.
<a href="http://genome.cshlp.org/content/12/4/656.full" target="_blank">
BLAT - the BLAST-like alignment tool</a>.
<em>Genome Res.</em> 2002 Apr;12(4):656-64.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/11932250" target="_blank">11932250</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC187518/" target="_blank">PMC187518</a>
</p>

<p>
Pruitt KD, Tatusova T, Maglott DR.
<a href="http://nar.oxfordjournals.org/content/33/suppl_1/D501.full" target="_blank">
NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</a>.
<em>Nucleic Acids Res</em>. 2005 Jan 1;33(Database issue):D501-4.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15608248" target="_blank">15608248</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC539979/" target="_blank">PMC539979</a>
</p>
