<h3>Description</h3>
<p>Expression data from <a href="http://www.rii.com">Rosetta Inpharmatics</a>.
See the paper "<a href="https://www.ncbi.nlm.nih.gov:80/entrez/query.fcgi?cmd=Retrieve&db=PubMed&list_uids=11237012&dopt=Abstract">Experimental Annotation of the Human Genome Using Microarray Technology</a>"
<i>Nature</i> Feb. 2001, vol 409 pp 922-7 for more
information. Briefly, Rosetta created DNA probes for each exon as
described by the Sanger center for the October 2000 draft of the
genome and used them to explore expression leves over 69 different
experiments. As in the original paper exons are labeled according to
contig name, relative position in the contig, and whether they were
predicted (pe) or confirmed (true->te) exons at the time of
publication. For example, AC000097_256_te is the 256th exon on
AC000097 predicted by Genescan which was confirmed
independently. Hybridization names refer to the sources of the two
mRNA populations used for the experiment.
<b>Please note:</b> in the browser window the hybridization names
are too long to fit and have been abbreviated. Also, the ratios
were inverted as of Feb 12, 2002 to conform with standard microarray
conventions of having the experimental sample in the red (cy5) channel
and the reference sample in the green (cy3) channel.

<h3>Display Options</h3>
The track can be configured with a few different options:

<br><b>Reference Sample: </b> This option is only valid when the track is displayed in
full. It determines how the 69 different experiments are displayed. The
options are:
<ul>
<li><b>All Experiments: </b> Display the data for each experiment one
   row per experiment.</li>
<li><b>Common Reference and Other:</b> Displays a summary value for all the
   experiments done with a common pooled reference sample, and another summary value for 
   the rest of the experiments.</li>
<li><b>Common Pool:</b> Only displays experiments which were performed using a 
   common pooled reference sample.</li>
<li><b>Other:</b> Only displays experiments which were not performed using a common
   pooled reference sample.</li>
</ul>

<b>Exons Shown: </b> Probes on the microarrays correspond to gene
predictions on chromosome 22, some of which were confirmed by known
genes, others are predictions. This option determines whether data are
shown for probes corresponding to confirmed, predicted, or all exons
are shown.

<br><b>Color Scheme: </b> Data are presented using two color false
display. By default the Brown/Botstein colors of red -> positive log
ratio, green -> negative log ratio are used.  However, blue can be
substituted for green for those who are color blind. Gray values
indicate missing data.  Please note that due to technical limitations
the details page will have many more color shades possible than those used
on the browser image and thus may not match exactly.

<h3>Details Page</h3>
On the details page the probes presented correspond to those contained
in window range seen on the Genome Browser, the exon probe selected is highlighted
in blue. The detail display table is actually an average of many data
points. It is possible to see the full data for each experiment
graphically by selecting the check-boxes for the experiments of interest
and clicking the submit value button.
