<H2>Description</H2>
<P>
The SIB Genes track is a transcript-based set of gene predictions based
on data from RefSeq and EMBL/GenBank. Genes all have the support of at
least one GenBank full length RNA sequence, one RefSeq RNA, or one spliced
EST.  The track includes both protein-coding and non-coding transcripts.
The coding regions are predicted using
<A HREF="http://estscan.sourceforge.net" TARGET="_BLANK">ESTScan</A>.</P>

<H2>Display Conventions and Configuration</H2>
<P>
This track in general follows the display conventions for
<A HREF="../goldenPath/help/hgTracksHelp.html#GeneDisplay">gene prediction
tracks</A>.  The exons for putative non-coding genes and untranslated regions 
are represented by relatively thin blocks while those for coding open 
reading frames are thicker.</P>
<P>
This track contains an optional codon coloring
feature that allows users to quickly validate and compare gene predictions.
To display codon colors, select the <em>genomic codons</em> option from the
<em>Color track by codons</em> pull-down menu.  Go to the
<A HREF="../goldenPath/help/hgCodonColoring.html">Coloring Gene Predictions and
Annotations by Codon</A> page for more information about this feature. </P>
<P>Further information on the predicted transcripts can be found on the
<A HREF="https://ccg.vital-it.ch/tromer/" TARGET="_BLANK">Transcriptome Web
interface</A>.</P>


<H2>Methods</H2>
<P>
The SIB Genes are built using a multi-step pipeline:  
<OL>
<LI>RefSeq and GenBank RNAs and ESTs are aligned to the genome with
<A HREF="http://sibsim4.sourceforge.net" TARGET="_BLANK">SIBsim4</A>, keeping 
only the best alignments for each RNA.
<LI>Alignments are broken up at non-intronic gaps, with small isolated 
fragments thrown out.
<LI>A splicing graph is created for each set of overlapping alignments. This
graph has an edge for each exon or intron, and a vertex for each splice site,
start, and end.  Each RNA that contributes to an edge is kept as evidence for
that edge.
<LI>The graph is traversed to generate all unique transcripts. The traversal is 
guided by the initial RNAs to avoid a combinatorial explosion in alternative 
splicing.
<LI>Protein predictions are generated.
</OL>

<H2>Credits</H2>
<P>
The SIB Genes track was produced on the <A
HREF="http://www.vital-it.ch" TARGET="_BLANK">Vital-IT</A> high-performance 
computing platform
using a computational pipeline developed by Christian Iseli with help from
colleagues at the <A HREF="https://www.ludwigcancerresearch.org/" TARGET="_BLANK">Ludwig Institute
for Cancer
Research</A> and the <A HREF="https://www.sib.swiss/" TARGET="_BLANK">Swiss Institute 
of Bioinformatics</A>.  It is based on data from NCBI <a
href="https://www.ncbi.nlm.nih.gov/refseq/" TARGET="_blank">RefSeq</a> and <a
href="https://www.ncbi.nlm.nih.gov/genbank/"
target="_blank">GenBank</a>/<a href="https://www.embl.org/"
target="_blank">EMBL</a>.  Our
thanks to the people running these databases and to the scientists worldwide
who have made contributions to them.</P>

<H2>References</H2>
<p>
Benson DA, Karsch-Mizrachi I, Lipman DJ, Ostell J, Wheeler DL.
<a href="https://academic.oup.com/nar/article/32/suppl_1/D23/2505202/GenBank-update"
target="_blank">GenBank: update</a>.
<em>Nucleic Acids Res</em>. 2004 Jan 1;32(Database issue):D23-6.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/14681350" target="_blank">14681350</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC308779/" target="_blank">PMC308779</a>
</p>
