<H2>Description</H2>
<P>
  This track consolidates all the Simple Nucleotide Polymorphisms (SNPs) into
  a single track.  This represents data from dbSnp and commercially-available 
  genotyping arrays.
</P>
<P>
  Please be aware that some mapping inconsistencies are known to exist in 
  the dbSnp data set. If you encounter information that seems incorrect on 
  the details page for a variant, we advise you to verify the record information
  on the dbSnp website using the provided link. In some
  known instances, the size of the variant does not match the size of its 
  genomic location; UCSC is working with dbSnp to correct these errors in
  the data set. 
</P>
<H2>Interpreting and Configuring the Graphical Display</H2>
<P>
  Variants are shown as single tick marks at most zoom levels.
  When viewing the track at or near base-level resolution, the displayed
  width of the SNP corresponds to the width of the variant in the reference
  sequence. Insertions are indicated by a single tick mark displayed between
  two nucleotides, single nucleotide polymorphisms are displayed as the width 
  of a single base, and multiple nucleotide variants are represented by a 
  block that spans two or more bases.
</P>
<P>
  When the start coordinate for a SNP is shown as chromStart = chromEnd+1 on 
  the SNP's details page, this is generally not an 
  error; rather, it indicates that the variant is an insertion at this genomic
  position.  In these instances, the location type will be set to 
  &quot;between&quot;. Note that insertions are represented as chromStart = 
  chromEnd in the <em>snp</em> table accessible from the Table Browser 
  or downloads server, due to the half-open zero-based representation of
  data in the underlying database. 
</P>
<P>
  The colors of variants in the display may be changed to highlight
  their source, molecule type, variant class, validation status, or
  functional classification. Variants can be excluded from the display
  based on these same criteria or if they fall below the
  user-specified minimum <A name="AvHet"></A>
  <A
  HREF="https://www.ncbi.nlm.nih.gov/SNP/Hetfreq.html" target="_blank">
  average heterozygosity</A>. The track configuration options are
  located at the top of the <A HREF="hgTrackUi?g=snp"> SNPs track
  description page</A>.  By default variants are colored by functional
  classification, with SNPs likely to cause a phenotype in red
  (non-synonymous and splice site mutations).
</P>
<P>
  The following configuration categories reflect the following definitions defined
  in the document type definition (DTD) that describes the 
  dbSnp XML format.  
</P>
  <UL>
    <LI>
      <A name="Source"></A>
      <B>Source</B>: Origin of this data<BR>
      <UL>
        <LI><B>dbSnp</B> - From the current build of dbSnp
        <LI><B>Affymetrix Genotyping Array 10K</B> - SNPs on the commercial array
        <LI><B>Affymetrix Genotyping Array 10K v2</B> -  SNPs on the commercial array
        <LI><B>Affymetrix Genotyping Array 50K HindIII</B> -  SNPs on the commercial array
        <LI><B>Affymetrix Genotyping Array 50K XbaI</B> -  SNPs on the commercial array
      </UL>
    </LI>
    <LI>
      <A name="MolType"></A>
      <B>Molecule Type</B>: Sample used to find this variant<BR>
      <UL>
        <LI><B>Unknown</B> - sample type not known
        <LI><B>Genomic</B> - variant discovered using a genomic template
        <LI><B>cDNA</B> - variant discovered using a cDNA template
        <LI><B>Mitochondrial</B> - variant discovered using a mitochondrial template
        <LI><B>Chloroplast</B> - variant discovered using a chloroplast template
      </UL>
    </LI>
    <LI>
      <A name="Class"></A>
      <B>Variant Class</B>: Variant classification<BR>
      <UL>
        <LI><B>Unknown</B> - no classification provided by data contributor
        <LI><B>Single Nucleotide Polymorphism</B> - single nucleotide 
	    variation: alleles of length = 1 and from set of {A,T,C,G}
        <LI><B>Insertion/deletion</B> - insertion/deletion variation: alleles 
	    of different length or include '-' character
        <LI><B>Heterozygous</B> - heterozygous (undetermined) variation: 
	    allele contains string '(heterozygous)'
        <LI><B>Microsatellite</B> - microsatellite variation: allele string 
	    contains numbers and '(motif)' pattern
        <LI><B>Named</B> - insertion/deletion of named object (length unknown)
        <LI><B>No Variation</B> - no variation asserted for sequence
        <LI><B>Mixed</B> - mixed class
        <LI><B>Multiple Nucleotide Polymorphism</B> - alleles of the same 
	    length, length > 1, and from set of {A,T,C,G}
      </UL>
    </LI>
    <LI>
      <A name="Valid"></A>
      <B><A HREF="https://www.ncbi.nlm.nih.gov/SNP/snp_legend.cgi?legend=validation" 
	target="_blank">Validation Status</A></B>: Method used to validate
	the variant (<I>each variant may be validated by more than one method</I>)<BR>
        <UL>
        <LI><B>Unknown</B> - no validation has been reported for this refSNP
        <LI><B>Other Population</B> - at least one ss in cluster was validated
	    by independent assay
        <LI><B>By Frequency</B> - at least one subsnp in cluster has frequency
	    data submitted
        <LI><B>By Cluster</B> - cluster has 2+ submissions, with 1+ 
	    submissions assayed with a non-computational method
        <LI><B>By 2 Hit/2 Allele</B> - all alleles have been observed in 2+ 
	    chromosomes
        <LI><B>By HapMap</B> - validated by HapMap project
        <LI><B>By Genotype</B> - at least one genotype reported for this refSNP
      </UL>
    </LI>
    <LI>
      <A name="Func"></A>
      <B><A HREF="https://www.ncbi.nlm.nih.gov/SNP/snp_legend.cgi?legend=snpFxnColor" 
	target="_blank">Function</A></B>: Predicted functional role (<I>each 
	variant may have more than one functional role</I>)<BR>
      <UL>
        <LI><B>Unknown</B> - no known functional classification
        <LI><B>Locus Region</B> - variation in region of gene, but not in 
	    transcript
        <LI><B>Coding</B> - variation in coding region of gene, assigned if 
	    allele-specific class unknown
        <LI><B>Coding - Synonymous</B> - no change in peptide for allele with 
	    respect to contig seq
        <LI><B>Coding - Non-Synonymous</B> - change in peptide with respect to
	    contig sequence
        <LI><B>mRNA/UTR</B> - variation in transcript, but not in coding 
	    region interval
        <LI><B>Intron</B> - variation in intron, but not in first two or last 
	    two bases of intron
        <LI><B>Splice Site</B> - variation in first two or last two bases of 
	    intron
        <LI><B>Reference</B> - allele observed in reference contig sequence
        <LI><B>Exception</B> - variation in coding region with exception 
	    raised on alignment. This occurs when protein with gap in sequence is 
	    aligned back to contig sequence. Variations that are on the 3' side 
	    of the gap have undefined functional inference.
      </UL>
    </LI>
    <LI>
      <A name="LocType"></A>
      <B>Location Type</B>: Describes how a segment of the reference assembly 
	must be altered to represent the variant SNP allele<BR>
      <UL>
        <LI><B>Unknown</B> - undefined or error
        <LI><B>Range</B> - a range of two or more bases in the reference 
	    assembly must be altered. This occurs, for example, when the variant
	    allele is a deletion of two or more bases relative to the allele 
	    represented by the reference assembly.
        <LI><B>Exact</B> - one base in the reference assembly must be altered.
	    This occurs when the variant allele is a single-base substitution
	    relative to the reference genome or when the variant allele is a 
	    deletion of a single base.
        <LI><B>Between</B> - no reference assembly bases must be altered.
	    This occurs when the variant allele is an insertion of one or more
	    bases relative to the allele represented by the reference assembly.
      </UL>
    </LI>
  </UL>

<A name="LSSNP"></A>
<H2>Large Scale SNP Annotation at UCSF</H2>
<P>
  LS-SNP is a database of functional and structural SNP annotations
  with links to protein structure models.  Annotations are based on a
  variety of features extracted from protein structure, sequence, and
  evolution.  Currently only coding non-synonomous SNPs are included.
  <A HREF="http://modbase.compbio.ucsf.edu/LS-SNP/" target=_BLANK>LS-SNP at UCSF</A>.
</P>

<H2>Data Filtering</H2>
<P>
  The SNPs in this track include all known polymorphisms available in the
  current build of <A HREF="https://www.ncbi.nlm.nih.gov/SNP/"
  TARGET=_blank>dbSnp</A> that can be mapped against the current assembly. 
  The version of dbSnp from which these data were obtained can be found in the
  SNP track entry in the Genome Browser 
  <A HREF="/goldenPath/releaseLog.html">release log</A>.
</P>
<P>
  There are two reasons that some variants may not be mapped and/or
  annotated in this track:
</P>
  <UL>
    <LI> 
      Submissions are completely masked as repetitive elements.
      These are dropped from any further computations. This set of
      reference SNPs is found in chromosome &quot;rs_chMasked&quot; on
      the <A HREF="ftp://ftp.ncbi.nih.gov/snp/" TARGET=_blank>dbSNP
      ftp site</A>.
    </LI>
    <LI>
      Submissions are defined in a cDNA context with extensive
      splicing. These SNPs are typically annotated on refSeq mRNAs
      through a separate annotation process. Effort is being made to
      reverse map these variations back to contig coordinates, but
      that has not been implemented. For now, you can find this set of
      variations in &quot;rs_chNotOn&quot; on the <A
      HREF="ftp://ftp.ncbi.nih.gov/snp/" TARGET=_blank>dbSNP ftp
      site</A>.  
    </LI>
  </UL>
<P>
  The heuristics for the non-SNP variations (i.e. named elements and
  short tandem repeats (STRs)) are quite conservative; therefore, some of 
  these are probably lost. This approach was chosen to avoid false 
  annotation of variation in inappropriate locations.
</P>

<H2>Credits and Data Use Restrictions</H2>
<P>
  Thanks to the <A HREF="http://snp.cshl.org/" TARGET=_blank>SNP
  Consortium</A> and <A HREF="https://www.ncbi.nlm.nih.gov"
  TARGET=_blank>NIH</A> for providing the public data, which are
  available from <A HREF="https://www.ncbi.nlm.nih.gov/SNP"
  TARGET=_blank>dbSnp</A> at NCBI.
</P>
<P>
  Thanks to <A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis.html" TARGET=_blank>Affymetrix, Inc.</A> 
  for developing the genotyping arrays. Please see the 
  <A HREF="http://www.affymetrix.com/site/terms.affx"
  TARGET=_blank>Terms and Conditions</A> page on the Affymetrix
  website for restrictions on the use of their data.
  For more details on the Affymetrix genotyping assay, see the supplemental 
  information on the 
  <A HREF="http://tools.thermofisher.com/content/sfs/brochures/10k2_datasheet.pdf"
  TARGET=_blank>Affymetrix 10K SNP</A> and 
  <A HREF="https://www.thermofisher.com/us/en/home/life-science/microarray-analysis/microarray-data-analysis/microarray-analysis-sample-data/genotyping-allele-frequency-data.html" 
  TARGET=_blank>Affymetrix Genotyping Array</A> products. Additional 
  information, including genotyping data, is available on those pages.
</P>
<P>
  Karchin, R., Diekhans, M., Kelly, L., Thomas, D.J., Pieper, U., Eswar, N.,
  Haussler, D. and Sali, A.
  <A HREF="https://academic.oup.com/bioinformatics/article/21/12/2814/268096"
  TARGET=_blank>LS-SNP: large-scale annotation of coding non-synonymous SNPs based on 
  multiple information sources</A>. 
  <I>Bioinformatics</I> <B>21</B>:2814-2820; April 12, 2005.
</P>
