<h2>Description</h2>

<p>
This track shows protein sequence annotations from the <a
href="https://www.uniprot.org/" target="_blank">UniProt/SwissProt</A> database,
mapped to genomic coordinates.
The data has been curated from scientific publications by the UniProt staff.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of UniProt/SwissProt annotations are labeled with a short name for
the type of annotation (e.g. &quot;glyco&quot;, &quot;disul&quot;, &quot;signal pep&quot; etc.). 
A click on them shows the full annotation and provides a link to the UniProt/SwissProt
record for all details.  </p>

<p>
Mouse over a mutation to see the UniProt comments.
</p>

<p>
Modified residues are highlighted in light blue, transmembrane regions in blue,
glycosylation sites in yellow, disulfide bonds in grey, topological domains in
red.
</p>

<h2>Methods</h2>

<p>
UniProt sequences were aligned to UCSC Genes / Gencode sequences first with
BLAT, then lifted to genome positions with pslMap.  UniProt variants were
parsed from the UniProt XML file.  The variants were then mapped to the genome
through the alignment using the pslMap program.  This mapping approach
draws heavily on the <A HREF="http://modbase.compbio.ucsf.edu/LS-SNP/"
TARGET="_BLANK">LS-SNP</A> pipeline by Mark Diekhans. The complete script is
part of the kent source tree and is located in src/hg/utils/uniprotMutations. 
</p>

<h2>Data Access</h2>

<p>
The raw data can be explored interactively with the
<a href="../cgi-bin/hgTables">Table Browser</a>, or the
<a href="../cgi-bin/hgIntegrator">Data Integrator</a>.
For automated analysis, the genome annotation is stored in a bigBed file that 
can be downloaded from the
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/" target="_blank">download server</a>.
The file for this track is called <tt>spAnnot.bb</tt>. Individual
regions or the whole genome annotation can be obtained using our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool can also be used to obtain only features within a given range, for example:
<br> 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/bbi/spAnnot.bb -chrom=chr6 -start=0 -end=1000000 stdout</tt> 
<br>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information. 
</p>

<h2>Credits</h2>

<p>
This track was created by Maximilian Haeussler, with advice from Mark Diekhans and Brian Raney.
Thanks to UniProt for making all data available for download.
</p>

<h2>References</h2>

<p>
UniProt Consortium.
<a href="https://academic.oup.com/nar/article/40/D1/D71/2903687/Reorganizing-the-protein-space-at-
the-Universal" target="_blank">
Reorganizing the protein space at the Universal Protein Resource (UniProt)</a>.
<em>Nucleic Acids Res</em>. 2012 Jan;40(Database issue):D71-5.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22102590" target="_blank">22102590</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245120" target="_blank">PMC3245120</a>
</p>

<p>
Yip YL, Scheib H, Diemand AV, Gattiker A, Famiglietti LM, Gasteiger E, Bairoch A.
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.20021" target="_blank">
The Swiss-Prot variant page and the ModSNP database: a resource for sequence and structure
information on human protein variants</a>.
<em>Hum Mutat</em>. 2004 May;23(5):464-70.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15108278" target="_blank">15108278</a>
</p>
