<H2>Description</H2>
<P>
The 
<A HREF="http://supfam.mrc-lmb.cam.ac.uk/SUPERFAMILY" TARGET=_blank>Superfamily</A> 
track shows proteins having homologs with known structures or functions.</P>
<P>
Each entry on the track shows the coding region of a gene (based on Ensembl gene predictions).
In full display mode, the label for an entry consists of the names of 
all known protein domains encoded by this gene.  This 
usually contains structural and/or functional descriptions that provide valuable 
information to help users get a quick grasp of the biological significance of the 
gene.</P>

<H2>Methods</H2>
<P>
Data are downloaded from the Superfamily server.
Using the cross-reference between Superfamily entries and Ensembl gene prediction 
entries and their alignment to the appropriate genome, the associated data are 
processed to generate a simple BED format track.</P>
<H2>Credits</H2>
<P>
Superfamily was developed by
<A HREF="http://supfam.mrc-lmb.cam.ac.uk/gough/" TARGET = _blank>Julian
Gough</a> at the <a href="http://www2.mrc-lmb.cam.ac.uk/" TARGET=_blank>MRC Laboratory
of Molecular Biology</a>, Cambridge.</P>
<P>
Gough, J., Karplus, K., Hughey, R. and
Chothia, C. (2001). &quot;<a href="http://www.mrc-lmb.cam.ac.uk/genomes/julian/gough_jmb_supfam_2001.pdf" TARGET=_blank>Assignment of Homology to Genome Sequences using a
Library of Hidden Markov Models that Represent all Proteins of Known Structure</A>&quot;. 
<em>J. Mol. Biol.</em>, <b>313(4)</b>, 903-919.</P>

