<h2>Description</h2>

<p>
This track shows protein sequence annotations from the <a
href="http://www.crapome.org/" target="_blank">Crapome</A> database.
The Crapome database contains genomic annotations of ... 
</p>

<p>
This track contains the following subtracks:
</p> 
<table class="stdTbl">
  <tr>
    <th>Track Name</th>
    <th>Description</th>
  </tr>
  <tr>
    <td>light crap</td>
    <td>light contaminants with a c-value under 0.01</td>
</tr>
  <tr>
    <td>heavy crap</td>
    <td>heavy contaminants with a c-value in excess of 0.01</td>
</tr>
</table>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of Crapome annotations are labeled with ...
Annotations are shown in 
<span style="color: rgb(0,150,250)">light blue</span> when xxxx
</p>

<p>
Mouse over a feature to see xxx
</p>

<h2>Methods</h2>

<p>
Crapome sequences were aligned to the UCSC genome with BLAT, ...
</p>

<p>
A shell script log that documents how this track was built can be found on 
<a href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/makeDb/doc/hg38/gtex.txt">GitHub</a>.
</p>

<h2>Data Access</h2>

<p>
The raw data can be explored interactively with the
<a href="../cgi-bin/hgTables">Table Browser</a>, or the
<a href="../cgi-bin/hgIntegrator">Data Integrator</a>.
For automated analysis, the genome annotation is stored in a bigBed file that 
can be downloaded from the
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/crapome/" target="_blank">download server</a>.
The exact filenames can be found in our 
<a href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/makeDb/trackDb/trackDb.uniprot.ra">track configuration file</a>. 
Annotations can be converted to ASCII text by our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool can also be used to obtain only features within a given range, for example:
<br> 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/crapome/crapome.bb -chrom=chr6 -start=0 -end=1000000 stdout</tt> 
<br>

This track is updated every month. The MySQL table hgFixed.trackVersion
contains the name of the currently available data on the website. Older
versions of the data files can be downloaded from the <a
href="http://hgdownload.cse.ucsc.edu/goldenPath/$db/archive/">archive
folder</a> of our downloads server.  <br>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information. 
</p>

<h2>Credits</h2>

<p>
This track was created by X at UCSC, Y and Z. Special thanks to ZZZ for making the data available.</p>

<h2>References</h2>

<!-- create references on hgwdev with the tool getTrackReferences <pmid> --> 
