<h2>Description</h2>

<p>
This track shows protein sequence annotations from the <a
href="https://www.uniprot.org/" target="_blank">UniProt/SwissProt</A> database,
mapped to genomic coordinates. It also shows how the protein sequences in this database 
map to the genome.
The data has been curated from scientific publications by the UniProt/SwissProt staff.
The annotations are divided into multiple subtracks, based on their &quot;feature type&quot; in UniProt:
</p> 

<table class="stdTbl">
  <tr>
    <th>Track Name</th>
    <th>Description</th>
  </tr>
  <tr>
    <td>UCSC Alignment, SwissProt</td>
    <td>Protein sequences from SwissProt mapped onto the genome. All other
        tracks are (start,end) annotations mapped using this track.</td> </tr>
  <tr>
    <td>UCSC Alignment, TrEMBL</td>
    <td>Protein sequences from TrEMBL mapped onto the genome. All other tracks
        are (start,end) annotations mapped using this track. This track is
hidden by default. To show it, click its checkbox on the track description
page.</td> </tr>
  <tr>
    <td>UniProt Signal Peptides</td>
    <td>Regions found in proteins destined to be secreted, generally cleaved from mature protein.</td>
  </tr>
  <tr>
    <td>UniProt Extracellular Domains</td>
    <td>Protein domains with the comment &quot;Extracellular&quot;.</td>
  </tr>
  <tr>
    <td>UniProt Transmembrane Domains</td>
    <td>Protein domains of the type &quot;Transmembrane&quot;.</td>
  </tr>
  <tr>
    <td>UniProt Cytoplasmic Domains</td>
    <td>Protein domains with the comment &quot;Cytoplasmic&quot;.</td>
  </tr>
  <tr>
    <td>UniProt Polypeptide Chains</td>
    <td>Polypeptide chain in mature protein after post-processing.</td>
  </tr>
  <tr>
    <td>UniProt Domains</td>
    <td>Protein domains, zinc finger regions and topological domains.</td>
  </tr>
  <tr>
    <td>UniProt Disulfide Bonds</td>
    <td>Disulfide bonds.</td>
  </tr>
  <tr>
    <td>UniProt Amino Acid Modifications</td>
    <td>Glycosylation sites, modified residues and lipid moiety-binding regions.</td>
  </tr>
  <tr>
    <td>UniProt Amino Acid Mutations</td>
    <td>Mutagenesis sites and sequence variants.</td>
  </tr>
  <tr>
    <td>UniProt Protein Primary/Secondary Structure Annotations</td>
    <td>Beta strands, helices, coiled-coil regions and turns.</td>
  </tr>
  <tr>
    <td>UniProt Sequence Conflicts</td>
    <td>Differences between Genbank sequences and the UniProt sequence.</td>
  </tr>
  <tr>
    <td>UniProt Repeats</td>
    <td>Regions of repeated sequence motifs or repeated domains.</td>
  </tr>
  <tr>
    <td>UniProt Other Annotations</td>
    <td>All other annotations</td>
  </tr>
</table>
<p>
For consistency, the subtrack &quot;UniProt/SwissProt Variants&quot; is a copy of the track
&quot;UniProt Variants&quot; in the track group &quot;Phenotype and Literature&quot;, or 
&quot;Variation and Repeats&quot;, depending on the assembly.
</p>

<h2>Display Conventions and Configuration</h2>

<p>
Genomic locations of UniProt/SwissProt annotations are labeled with a short name for
the type of annotation (e.g. &quot;glyco&quot;, &quot;disulf bond&quot;, &quot;Signal peptide&quot;
etc.). A click on them shows the full annotation and provides a link to the UniProt/SwissProt
record for more details. TrEMBL annotations are always shown in 
<span style="color: rgb(0,150,250)">light blue</span>, except in the Signal Peptides,
Extracellular Domains, Transmembrane Domains, and Cytoplamsic domains subtracks.</p>

<p>
Mouse over a feature to see the full UniProt annotation comment. For variants, the mouse over will
show the full name of the UniProt disease acronym.
</p>

<p>
The subtracks for domains related to subcellular location are sorted from outside to inside of 
the cell: <span style="color: rgb(255,0,150)">Signal peptide</span>, 
<span style="color: rgb(0,150,255)">extracellular</span>, <span style="color: rgb(0,150,0)">
transmembrane</span>, and <span style="color: rgb(255,150,0)">cytoplasmic</span>.
</p>

<p>
In the &quot;UniProt Modifications&quot; track, lipoification sites are highlighted in 
<span style="color: rgb(12,12,120)">dark blue</span>, glycosylation sites in 
<span style="color: rgb(0,100,100)">dark green</span>, and phosphorylation in 
<span style="color: rgb(200,200,0)">light green</span>.</p>

<p>
Duplicate annotations are removed as far as possible: if a TrEMBL annotation
has the same genome position and same feature type, comment, disease and
mutated amino acids as a SwissProt annotation, it is not shown again. Two
annotations mapped through different transcripts but with the same genome
coordinates are only shown once.  </p>

<p>Note that only for the human hg38 assembly and SwissProt annotations, there
also is a <a
href="hgTracks?db=hg38&hubUrl=ftp://ftp.uniprot.org/pub/databases/uniprot/current_release/knowledgebase/genome_annotation_tracks/UP000005640_9606_hub/hub.txt">public
track hub</a> prepared by UniProt itself, with 
genome annotations maintained by UniProt using their own mapping
method based on those Gencode/Ensembl gene models that are annotated in UniProt
for a given protein.</p>

<h2>Methods</h2>

<p>
UniProt sequences were aligned to UCSC/Gencode transcript sequences first with
BLAT, filtered with pslReps (93% query coverage, within top 1% score), lifted
to genome positions with pslMap and filtered again.  UniProt annotations were
obtained from the UniProt XML file.  The annotations were then mapped to the
genome through the alignment using the pslMap program.  This mapping approach
draws heavily on the <A HREF="http://modbase.compbio.ucsf.edu/LS-SNP/"
TARGET="_BLANK">LS-SNP</A> pipeline by Mark Diekhans. For human and mouse, the
alignments were filtered by retaining only proteins annotated with
a given transcript in the Genome Browser table kgXref. Like all Genome Browser
source code, the main script used to build this track can be found on 
<a href="https://github.com/ucscGenomeBrowser/kent/tree/master/src/hg/utils/otto/uniprot/doUniprot">github</a>.
</p>

<h2>Data Access</h2>

<p>
The raw data can be explored interactively with the
<a href="../cgi-bin/hgTables">Table Browser</a>, or the
<a href="../cgi-bin/hgIntegrator">Data Integrator</a>.
For automated analysis, the genome annotation is stored in a bigBed file that 
can be downloaded from the
<a href="http://hgdownload.soe.ucsc.edu/gbdb/$db/uniprot/" target="_blank">download server</a>.
The exact filenames can be found in the 
<a href="https://github.com/ucscGenomeBrowser/kent/blob/master/src/hg/makeDb/trackDb/trackDb.uniprot.ra">track configuration file</a>. 
Annotations can be converted to ASCII text by our tool <tt>bigBedToBed</tt>
which can be compiled from the source code or downloaded as a precompiled
binary for your system. Instructions for downloading source code and binaries can be found
<a href="http://hgdownload.soe.ucsc.edu/downloads.html#utilities_downloads">here</a>.
The tool can also be used to obtain only features within a given range, for example:
<br> 
<tt>bigBedToBed http://hgdownload.soe.ucsc.edu/gbdb/$db/uniprot/unipStruct.bb -chrom=chr6 -start=0 -end=1000000 stdout</tt> 
<br>
This track is updated every month. The MySQL table hgFixed.trackVersion
contains the name of the currently available data on the website. Older
versions of the data files can be downloaded from the <a
href="http://hgdownload.cse.ucsc.edu/goldenPath/$db/archive/">archive
folder</a> of our downloads server.  <br>
Please refer to our
<a href="https://groups.google.com/a/soe.ucsc.edu/forum/#!forum/genome">mailing list archives</a>
for questions, or our
<a href="../FAQ/FAQdownloads.html#download36">Data Access FAQ</a>
for more information. 
</p>

<h2>Credits</h2>

<p>
This track was created by Maximilian Haeussler at UCSC, with help from Chris
Lee, Mark Diekhans and Brian Raney, feedback from the UniProt staff and Alejo
Mujica, Regeneron Pharmaceuticals.  Thanks to UniProt for making all data
available for download.
</p>

<h2>References</h2>

<p>
UniProt Consortium.
<a href="https://academic.oup.com/nar/article/40/D1/D71/2903687/Reorganizing-the-protein-space-at-
the-Universal" target="_blank">
Reorganizing the protein space at the Universal Protein Resource (UniProt)</a>.
<em>Nucleic Acids Res</em>. 2012 Jan;40(Database issue):D71-5.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/22102590" target="_blank">22102590</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3245120/" target="_blank">PMC3245120</a>
</p>

<p>
Yip YL, Scheib H, Diemand AV, Gattiker A, Famiglietti LM, Gasteiger E, Bairoch A.
<a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/humu.20021" target="_blank">
The Swiss-Prot variant page and the ModSNP database: a resource for sequence and structure
information on human protein variants</a>.
<em>Hum Mutat</em>. 2004 May;23(5):464-70.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/15108278" target="_blank">15108278</a>
</p>
