<h2>Description</h2>

<p>Bidirectional promoters are the regulatory regions that fall between
pairs of genes, where the 5' ends of the genes within a pair are positioned
in close proximity to one another. This spacing facilitates the initiation
of transcription of both genes, creating two transcription forks that advance
in opposite directions. The formal definition of a bidirectional promoter
requires that the transcription initiation sites are separated by no more than
1,000 bp from one another. Using these criteria we have comprehensively
annotated the human and mouse genomes for the presence of bidirectional
promoters, using in silico approaches. The identification of these promoters
is contingent upon the presence of adjacent, oppositely oriented pairs of
genes, because few distinguishing features are available to uniquely identify
bidirectional promoters de novo. Genomic annotations used for our
identification phase include:
<ul>
<li><em>A</em>) <a href="hgTrackUi?g=knownGene">UCSC known genes</a>
annotations (items with score=800).
<li><em>B</em>) <a href="hgTrackUi?g=mrna">GenBank mRNA</a> annotations
(score=600).
<li><em>C</em>) <a href="hgTrackUi?g=intronEst">spliced ESTs</a> (score=400).
</ul>
The annotations for protein coding genes (<em>A</em>) are strongly supported
and therefore provide a high quality dataset for mapping bidirectional
promoters. In contrast, bidirectional promoters supported by spliced ESTs
(<em>C</em>) alone have varying levels of evidence, ranging from one
characterized transcript to hundreds of them. For this reason, the mRNA
annotation (<em>B</em>) from GenBank provides a stringent level of validation
for the start sites of the EST transcripts. As a large class of regulatory
sequences, bidirectional promoters exemplify a rich source of unexplored
biological information in the human genome. When compared to the mouse genome,
these promoters are identifiable as truly orthologous locations, being
maintained in regions of conserved synteny (including both genes and the
intervening promoter region) that have undergone no rearrangements since the
last common ancestor of mammals, and in some cases fish. We use this approach
to annotate orthologous bidirectional promoters in nonhuman species until
genomic annotations become available.
</p>

<h2>Methods</h2>

<h3>Assigning Orthologous Regions</h3>
<p>A multi-stage approach to mapping orthology at bidirectional promoters was
developed. Orthology assignments are strongest in coding regions. Therefore we
began by mapping single human genes regulated by bidirectional promoters from
the Known Genes annotations onto the mouse genome. Orthology assignments were
determined using the "chains and nets" data from the UCSC Human Genome Browser
mysql tables. Chains in the Genome Browser represent sequences of gapless
aligned blocks. Nets provide a hierarchical ordering of those chains. Level 1
chains contain the longest, best-scoring sequence chains that span any
selected region. Subsequent levels in the net represent the results of
rearrangements, duplications, insertions and deletions that may have disrupted
the presence of conserved synteny derived from an ancestral sequence.
</p>

<h3>Confirming Orthologous Genes</h3>
<p>After determining the orthology assignments using the UCSC chains and nets
data, we used the Known Gene annotations or spliced ESTs to search the identity
of genes within the corresponding region. Known Genes represent protein-coding
genes and therefore can be verified by chains and nets alignments, followed by
confirmation of protein identity in both species. Spliced ESTs carry less
descriptive information than protein coding genes and therefore were validated
in the second species by their presence in an orthologous region, showing
conserved synteny of the two genes within a pair, and meeting the criteria of
less than 1,000 bp of intergenic distance between those transcripts. Our method
for mapping bidirectional promoters in spliced EST datasets is described in
more detail in a previous publication. If the program verified evidence for
orthology and conserved-syntenic gene arrangement, then the orthologous
bidirectional promoter was confirmed. After orthologous assignments were
confirmed for pairs of human genes, the reciprocal assignments were analyzed
from mouse to human.

Currently orthologous bidirectional promoter regions (that have been identified
using UCSC known genes) have been mapped in human, chimp, macaque, mouse, rat,
dog and cow genomes).
</p>

<h2>Credits</h2>

<p>These data were produced by Mary Q. Yang in the
<a href="https://www.genome.gov/12514761" title="https://www.genome.gov/12514761"
 TARGET=_BLANK>Elnitski lab</a> at NHGRI, NIH. (contact:
 <A HREF="mailto:&#101;&#108;&#110;&#105;&#116;&#115;ki&#64;&#109;&#97;&#105;l.n&#105;&#104;.g&#111;&#118;">
 &#101;&#108;&#110;&#105;&#116;&#115;ki&#64;&#109;&#97;&#105;l.n&#105;&#104;.g&#111;&#118;</A>)
 <!-- above address is elnitski at mail.nih.gov -->
</p>

<h2>References</h2>

<p>
Piontkivska H, Yang MQ, Larkin DM, Lewin HA, Reecy J, Elnitski L.
<a href="https://bmcgenomics.biomedcentral.com/articles/10.1186/1471-2164-10-189" target="_blank">
Cross-species mapping of bidirectional promoters enables prediction of unannotated 5' UTRs and
identification of species-specific transcripts</a>.
<em>BMC Genomics</em>. 2009 Apr 24;10:189.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/19393065" target="_blank">19393065</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2688522/" target="_blank">PMC2688522</a>
</p>

<p>
Yang MQ, Elnitski LL.
<a href="https://link.springer.com/chapter/10.1007%2F978-3-540-72031-7_33" target="_blank">
A computational study of bidirectional promoters in the human genome
</a>.
<em>Springer Lecture Series: Notes in Bioinformatics</em> 2007.
</p>

<p>
Yang MQ, Elnitski L.
Orthology of Bidirectional Promoters Enables Use of a Multiple Class Predictor for Discriminating
Functional Elements in the Human Genome.
Proceedings of the 2007 International Conference on Bioinformatics &amp; Computational Biology.
<!-- Re-add link when WORLDCOMP archives are updated:
http://www.world-academy-of-science.org/past_events/ws/conferences/2007

<a href="http://www.world-academy-of-science.org/worldcomp07/ws/BIOCOMP07" target="_blank">
Proceedings of the 2007 International Conference on Bioinformatics &amp; Computational Biology
</a>.
-->
pp. 218-228. 2007. ISBN: 1-60132-042-6.
</p>

<p>
Yang MQ, Koehly LM, Elnitski LL.
<a href="http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.0030072" target="_blank">
Comprehensive annotation of bidirectional promoters identifies co-regulation among breast and
ovarian cancer genes</a>.
<em>PLoS Comput Biol</em>. 2007 Apr 20;3(4):e72.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/17447839" target="_blank">17447839</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC1853124/" target="_blank">PMC1853124</a>
</p>

<p>
Yang MQ, Taylor J, Elnitski L.
<a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-9-S6-S9"
target="_blank">Comparative analyses of bidirectional promoters in vertebrates</a>.
<em>BMC Bioinformatics</em>. 2008 May 28;9 Suppl 6:S9.
PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/18541062" target="_blank">18541062</a>; PMC: <a
href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC2423431/" target="_blank">PMC2423431</a>
</p>

<H2>Data Release Policy</H2>

<P>Data users may freely use ENCODE data, but may not, without prior
consent, submit publications that use an unpublished ENCODE dataset until
nine months following the release of the dataset.  This date is listed in
the table<EM>metadata</EM> as <EM>dateUnrestricted</EM> and on the
download page.  The full data release policy for ENCODE is available
<A HREF="../ENCODE/terms.html" TARGET=_BLANK>here</A>.</P>
