<!--
<TABLE ALIGN=RIGHT BORDER=0 WIDTH=700>
  <TR><TD ALIGN=RIGHT>
    <A HREF="../images/M_capitatus.jpg">
    <IMG SRC="../images/M_capitatus.jpg" WIDTH=600 HEIGHT=300 ALT="M. capitatus"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Magnusiomyces capitatus</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      (photograph by Viktória Hodorová)
    </FONT>
  </TD></TR>
</TABLE>
-->

<P>
<B>Genome Browser assembly ID:</B> sapFunA1<BR>
<B>Sequencing/Assembly provider ID:</B> Comenius University in Bratislava / sapFunA1<BR>
<B>Assembly date:</B> December 2018<BR>
<!-- 
<B>Assembly accession:</B> <BR> 
<B>NCBI Genome ID:</B> <BR>
<B>NCBI Assembly ID:</B> <BR>
<B>BioProject ID:</B> 
-->

</P>
<HR>
<P>
<B>Search the assembly:</B>
<UL>
<LI>
<B>By position or search term: </B> Use the &quot;position or search term&quot;
box to find areas of the genome associated with many different attributes, such
as a specific chromosomal coordinate range; mRNA, EST, or STS marker names; or
keywords from the GenBank description of an mRNA.
<A HREF="../goldenPath/help/query.html">More information</A>, including sample
queries.
<LI>
<B>By gene name: </B> Type a gene name into the &quot;search term&quot; box,
choose your gene from the drop-down list, then press &quot;submit&quot; to go 
directly to the assembly location associated with that gene.
<A HREF="../goldenPath/help/geneSearchBox.html">More information</A>.
<LI>
<B>By track type: </B> Click the &quot;track search&quot; button
to find Genome Browser tracks that match specific selection criteria.
<A HREF="../goldenPath/help/trackSearch.html">More information</A>.
</UL>
</P>

<!--
<HR>
<P>
<B>Download sequence and annotation data:</B>
<UL>
<LI><A HREF="../goldenPath/help/ftp.html">Using rsync</A> (recommended)
<LI><A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg38/">Using FTP</A>
<LI><A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#human">Using HTTP</A>
<LI><A HREF="../goldenPath/credits.html#human_credits">Data use conditions and
restrictions</A>
<LI><A HREF="../goldenPath/credits.html#human_credits">Acknowledgments</A>
</UL>
</P>
<hr>
-->

<h3>Assembly Details</h3>
<p>
  The genome sequence of <em>S. fungicola</em> CBS 625.85
  strain was produced as a hybrid assembly, combining Illumina
  short reads (approx. 203x coverage) with
  MinION (approx. 148x coverage) long-read data. The
  assembly consists of 5 nuclear contigs (contig01-contig05) likely
  corresponding to whole chomosomes with
  total assembly size 20.2 Mbp.
  One contig represents the mitochondrial genome (mtDNA;
  size=33.0 kbp).

<p>
  Long reads were assembled by miniasm (Li 2016)
  followed by manual curation. The sequencing
  errors were corrected by Racon (Vaser et al. 2017)
  and Pilon (Walker et al. 2014).


  <h3>References</h3>

<p>
  Li H. (2016) Minimap and miniasm: fast mapping and de novo assembly for noisy long sequences. Bioinformatics 32:2103-2110. 

<p>
  Vaser R, Sović I, Nagarajan N, Šikić M. (2017) Fast and accurate de novo genome assembly from long uncorrected reads. Genome Res 27:737-746. 
  
<p>
  Walker, B.J., Abeel, T., Shea, T., Priest, M., Abouelliel, A., Sakthikumar, S., Cuomo, C.A., Zeng, Q., Wortman, J., Young, S.K. and Earl, A.M., 2014. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PloS one, 9(11), p.e112963.
