<TABLE ALIGN=RIGHT BORDER=0 WIDTH=700>
  <TR><TD ALIGN=RIGHT>
    <A HREF="../images/M_ingens.jpg">
    <IMG SRC="../images/M_ingens.jpg" WIDTH=600 HEIGHT=300 ALT="M. ingens"></A>
  </TD></TR>
  <TR><TD ALIGN=RIGHT> 
    <FONT SIZE=-1><em>Magnusiomyces ingens</em><BR> 
    </FONT>
    <FONT SIZE=-2> 
      (photograph by Viktória Hodorová)
    </FONT>
  </TD></TR>
</TABLE>

<P>
<B>Genome Browser assembly ID:</B> magIngA4<BR>
<B>Sequencing/Assembly provider ID:</B> Comenius University in Bratislava / magIngA4<BR>
<B>Assembly date:</B> October 2017<BR>
<!-- 
<B>Assembly accession:</B> <BR> 
<B>NCBI Genome ID:</B> <BR>
<B>NCBI Assembly ID:</B> <BR>
<B>BioProject ID:</B> 
-->

</P>
<HR>
<P>
<B>Search the assembly:</B>
<UL>
<LI>
<B>By position or search term: </B> Use the &quot;position or search term&quot;
box to find areas of the genome associated with many different attributes, such
as a specific chromosomal coordinate range; mRNA, EST, or STS marker names; or
keywords from the GenBank description of an mRNA.
<A HREF="../goldenPath/help/query.html">More information</A>, including sample
queries.
<LI>
<B>By gene name: </B> Type a gene name into the &quot;search term&quot; box,
choose your gene from the drop-down list, then press &quot;submit&quot; to go 
directly to the assembly location associated with that gene.
<A HREF="../goldenPath/help/geneSearchBox.html">More information</A>.
<LI>
<B>By track type: </B> Click the &quot;track search&quot; button
to find Genome Browser tracks that match specific selection criteria.
<A HREF="../goldenPath/help/trackSearch.html">More information</A>.
</UL>
</P>

<!--
<HR>
<P>
<B>Download sequence and annotation data:</B>
<UL>
<LI><A HREF="../goldenPath/help/ftp.html">Using rsync</A> (recommended)
<LI><A HREF="ftp://hgdownload.soe.ucsc.edu/goldenPath/hg38/">Using FTP</A>
<LI><A HREF="http://hgdownload.soe.ucsc.edu/downloads.html#human">Using HTTP</A>
<LI><A HREF="../goldenPath/credits.html#human_credits">Data use conditions and
restrictions</A>
<LI><A HREF="../goldenPath/credits.html#human_credits">Acknowledgments</A>
</UL>
</P>
<hr>
-->

<h3>Assembly Details</h3>
<p>
  The genome sequence of <em>M. ingens</em> NRRL Y-17630 (CBS
  521.90) strain was produced as a hybrid assembly, combining Illumina
  short reads (approx. 330x coverage) with PacBio RSII long reads (approx. 79x
  coverage). The
  assembly consists of 13 nuclear contigs (contig01-contig13; N50=2273.9 kbp) with total size 18.1 Mbp
  and one contig representing the mitochondrial genome (mtDNA;
  size=37.7 kbp).

<p>
  Long reads were assembled by Canu version 1.5 (Koren et al. 2017). The sequencing
  errors were corrected by two iterations of Pilon (Walker et al. 2014).
  Further corrections were applied in repeat-rich regions, by replacing
  appropriate parts of the repeat regions with contigs produced by
  SPAdes (Bankevich et al. 2012).

<h3>References</h3>
<p>
  Bankevich A., Nurk S., Antipov D., Gurevich A., Dvorkin M., Kulikov A. S., Lesin V., Nikolenko S., Pham S., Prjibelski A., Pyshkin A., Sirotkin A., Vyahhi N., Tesler G., Alekseyev M. A., Pevzner P. A. (2012) SPAdes: A New Genome Assembly Algorithm and Its Applications to Single-Cell Sequencing. Journal of Computational Biology, 19(5), 455-477.
<p>
  Koren, S., Walenz, B. P., Berlin, K., Miller, J. R., Bergman, N. H., & Phillippy, A. M. (2017). Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome research, gr-215087.
<p>
  Walker, B.J., Abeel, T., Shea, T., Priest, M., Abouelliel, A., Sakthikumar, S., Cuomo, C.A., Zeng, Q., Wortman, J., Young, S.K. and Earl, A.M., 2014. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PloS one, 9(11), p.e112963.
